test/router/router.test.lua

```diff
@@ -175,6 +175,8 @@ uuids = {}
 for uuid, _ in pairs(map) do table.insert(uuids, uuid) end
 uuids
 
+vshard.router.internal.static_router.failover_fiber:wakeup()
+
 --
 -- gh-69: aliases for router.call - callro and callrw.
 --
```

Comment:
Rerunned test 200 times locally, the problem seems to be gone. Don't really like that solution as I cannot see any relation to changes done in 2500190d11a74b394e424885b3aa156a523c0a7e and why specifically manual waking of failover fiber removes flakiness. Still investigating

Reply:
AFAICS, any increasing of the time, needed to execute the test after restarting of the `storage_1_a` seems to be enough to solve the flakiness of the test. However, this is not the root of the problem. 

I switched between storages in the first replicaset after inserting the tuple to the master and discovered, that in case of test failure there was no data on the replica, it was only on master. However, `vshard.storage.sync` said that the data is synchronized, which wasn't actually true. I described everything in commit message.